public/walkthroughs/walkthrough-2/walkthrough.adoc

```diff
@@ -0,0 +1,430 @@
+// Attributes
+:integreatly: Integreatly
+:imagesdir: /images
+:fuse-url: https://cluster-lfa3xlh.opentry.me
+:messaging-broker-url: https://console-enmasse.cluster-lfa3xlh.opentry.me/console/eval
+:openshift-url: http://employee.openshift.com/
+:sso-admin-url: https://console-keycloak.cluster-lfa3xlh.opentry.me/console/eval
+:messaging-service: Red Hat AMQ Online
+:messaging-service-version: 7.2
+:integration-service: Fuse
+:integration-service-version: 7.1
+:che-service: Che
+:launcher-service: Launcher
+:api-mgmt-service: 3Scale
+:AMQ-ProductLongName: Red Hat AMQ
+:AMQ-BrokerVersion: 7.2
+:Fuse-prodnamefull: Red Hat Fuse
+:Fuse-version: 7.1
+:3Scale-ProductName: Red Hat 3scale
+:3Scale-ProductVersion: 2.3
+:EnMasse-master-ProductLongName: EnMasse
+:walkthrough: Integrating API-driven applications
+
+= Integrating API-driven applications
+
+:context: integrating-api-driven-applications
+
+Integrate an API that provides flight information, ensuring that the API is protected and rate limited.
+
+APIs are the ideal channel for organizations who want to reach audiences whenever, wherever, and however they choose.
+This enables innovation, mobile and partner development.
+However, it also exposes the hosting infrastructure and organization.
+It is possible to protect and limit access to an API using {api-mgmt-service} as part of your integration.
+
+This walkthrough describes how to create such an integration, using OpenShift, {launcher-service}, {che-service} and {api-mgmt-service}.
+
+++++
+<img src="/images/wt2.png" class="img-responsive" alt="integration">
+++++
+
+[time=5]
+
+== Deploying the Fuse Aggregation App
+
+The Fuse Aggregation App provides information about flights, you deploy it using Launcher.
+
+[type=taskResource]
```

Comment:
At the moment we have to repeat the task resources for every task. I've added an action item to address this: `Allow walkthrough resources to be specified on the walkthrough overview page (e.g. by providing a [type=walkthroughResource] annotation)`